sdk/src/test/java/org/hiero/sdk/AccountCreateTransactionTest.java

```diff
@@ -17,6 +18,8 @@ public class AccountCreateTransactionTest {
     private static final PrivateKey unusedPrivateKey = PrivateKey.fromString(
             "302e020100300506032b657004220420db484b828e64b2d8f12ce3c0a0e93a0b8cce7af1bb8f39c97732394482538e10");
 
+    PrivateKeyECDSA privateKeyECDSA = new PrivateKeyECDSA(BigInteger.ONE, unusedPrivateKey.getChainCode());
```

Comment:
```suggestion
    private static final PrivateKey privateKeyED25519 = PrivateKey.fromString(
            "302e020100300506032b657004220420db484b828e64b2d8f12ce3c0a0e93a0b8cce7af1bb8f39c97732394482538e10");

    private static final PrivateKey privateKeyECDSA = PrivateKey.generateECDSA();
```
, delete `AccountCreateTransactionTest.snap` and run the tests again, should work.